doc/tutorials/11-ferrofluid/11-ferrofluid_part1.ipynb

```diff
@@ -88,13 +88,13 @@
     "\n",
```

Comment:
<p>i'd go for the relative change in energy which is system independent and pretty stable.</p><p>see https://github.com/espressomd/espresso/issues/3898#issuecomment-694352440</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/espressomd/espresso/pull/3919/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='39'/>